scripts/run_app.sh

```diff
@@ -13,6 +13,7 @@ export DM_API_AUTH_TOKEN=${DM_API_AUTH_TOKEN:=myToken}
 
 export DM_SUBMISSIONS_BUCKET=${DM_SUBMISSIONS_BUCKET:=digitalmarketplace-documents-dev-dev}
 export DM_COMMUNICATIONS_BUCKET=${DM_COMMUNICATIONS_BUCKET:=digitalmarketplace-documents-dev-dev}
+export DM_AGREEMENTS_BUCKET=${DM_AGREEMENTS_BUCKET:=digitalmarketplace-documents-dev-dev}
```

Comment:
This bucket currently only has stuff in from the functional tests.  Dumping stuff here is probably OK though, it shouldn't affect test runs.

The bucket `digitalmarketplace-dev-documents` seems to have been the preferred dumping ground for the rest of random stuff from running things locally up until now though.

@allait and/or @robyoung might have an opinion which is better? Although it probably doesn't matter either way really.